config/server-api-list.json

```diff
@@ -0,0 +1,14 @@
+{
+  "versions": [
+    {
+      "version": "v1.0",
```

Comment:
The goal of this is that every time we need to update the Rest API, we add new backend APIs without changing anything that was already established in a previous release - hence ensuring constant backwards compatibility. So for example, the v2.0 app might have a few APis that didn't exist in the 1.0 version, but only those are defined in that section and the majority of the apis default back to the v1.0 ones. We would know whether it is a new API altogether or a revised API that overrides a previous version based on whether or not there are keys overlapping in the "api-list" section. 

Note that even if a newer version of an API overrides an older one, both will continue to exist on the server side (again, for the sake of backwards compatibility).

Reply:
We would basically include the current version of this JSON file with each release of the app, which would help the app to figure out what the APIs' urls are that it wants to call. Does anyone have feedback on this general approach?